From cf8d8755aacac0be4285a516d9a041844c192f60 Mon Sep 17 00:00:00 2001
From: Xiongfeng Wang <wangxiongfeng2@huawei.com>
Date: Tue, 24 Aug 2021 21:18:09 +0800
Subject: [PATCH] X86/config: Enable CONFIG_USERSWAP
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: cf8d8755aacac0be4285a516d9a041844c192f60


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I40AXF
CVE: NA

--------------------------------------

Enable CONFIG_USERSWAP for openeuler_defconfig

Signed-off-by: Xiongfeng Wang <wangxiongfeng2@huawei.com>
Reviewed-by: tong tiangen <tongtiangen@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 6c225fc3f106..471d5c5faabb 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -8511,3 +8511,4 @@ CONFIG_ARCH_HAS_KCOV=y
 # end of Kernel hacking
 CONFIG_ETMEM_SCAN=m
 CONFIG_ETMEM_SWAP=m
+CONFIG_USERSWAP=y
-- 
2.26.2

